us.js

```diff
@@ -98,7 +106,7 @@
   };
 
   // https://github.com/kvz/phpjs/blob/master/functions/strings/metaphone.js
-  var metaphone = function (word, phones) {
+  us._metaphone = function (word, phones) {
```

Comment:
made this a 'private' member function so that it could be accessed while testing to help track down metaphone mismatches